apps/my-vehicles/types/i18next.d.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * If you want to enable locale keys typechecking and enhance IDE experience.
+ *
+ * Requires `resolveJsonModule:true` in your tsconfig.json.
+ *
+ * @link https://www.i18next.com/overview/typescript
+ */
+import "i18next";
+
+// resources.ts file is generated with `npm run toc`
```

Comment:
Should we remove this one or add the script to autogenerate `resources.ts`?

Reply:
I think we can remove the comment, it's enought.
45c8940